aclk/agent_cloud_link.c

```diff
@@ -922,7 +923,10 @@ void *aclk_query_main_thread(void *ptr)
     while (!netdata_exit) {
         if (unlikely(!aclk_metadata_submitted)) {
             aclk_metadata_submitted = ACLK_METADATA_CMD_QUEUED;
-            aclk_queue_query("on_connect", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT);
+            if (unlikely(aclk_queue_query("on_connect", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT))) {
```

Comment:
The other `on_connect` commands were changed but this one is not - what is the difference?

Reply:
What you mean ?  They all changed to check for return code / debug log

Reply:
Some of the calls to `aclk_queue_query` have changed from `on_connect` to `connector` and some have not?

Reply:
The first parameter `topic` is not used (except for incoming cloud commands) so it has changed to better align with the calling function. There is no difference in the functionality.